arch/arm/src/phy62xx/uart.c

```diff
@@ -878,10 +887,21 @@ static int pplus_uart_receive(struct uart_dev_s *dev, unsigned int *status)
 {
   uart_Ctx_t *priv = (uart_Ctx_t *)dev->priv;
   uint32_t data;
+  static uint8_t fifo_buf_store[UART_RX_BUFFER_SIZE];
+  static int buf_head;
+  static int buf_tail;
+  /* Put fifo data into loopback buffer */
+  for(int i=0;i<fifo_len_store;i++)
```

Comment:
add space after ;

Reply:
already fixed, will submit next release..

Reply:
you need fix the problem in this patch. Each patch should be in the good state before it get merge.